lib/ramble/docs/tutorials/configuring_a_scaling_study.rst

```diff
@@ -0,0 +1,245 @@
+suggested starting place for the experiments. To apply this to your workspace, use:
+
+
+.. code-block:: console
+
+    $ ramble workspace concretize
+
+This will fill out the ``spack`` dictionary within your workspace configuration
+file. After executing this command, your workspace configuration file might
+look like the following:
+
+.. code-block:: YAML
+
+    ramble:
+      variables:
+        processes_per_node: 4
+        n_ranks: '{processes_per_node}*{n_nodes}'
+        batch_submit: '{execute_experiment}'
+        mpi_command: mpirun -n {n_ranks}
+      applications:
+        wrfv4:
+          workloads:
+            CONUS_12km:
+              experiments:
+                scaling_{n_nodes}:
+                  variables:
+                    n_nodes: [1, 2, 4]
+      spack:
+        concretized: true
+        packages:
+          gcc9:
+            spack_spec: gcc@9.3.0
+          intel-mpi:
+            spack_spec: intel-mpi@2018.4.274
+            compiler: gcc9
+          wrfv4:
+            spack_spec: wrf@4.2 build_type=dm+sm compile_type=em_real nesting=basic ~chem
+              ~pnetcdf
+            compiler: gcc9
+        environments:
+          wrfv4:
+            packages:
+            - intel-mpi
+            - wrfv4
+
+At this point, you have fully described experiments that can be executed.
+However, your system might not have the correct compiler (and building a
+compiler could be costly). The ``gcc9`` package definition can be updated to
+refer to a compiler you already have on your system. These can be viewed using
+the ``spack compiler list`` command. Edit the ``gcc9`` package definition as
+you see fit, and make sure the ``gcc9`` references under ``intel-mpi`` and
+``wrfv4`` are updated appropriately as well.
+
+Execute Experiments
+-------------------
+
+Now that you have made the appropriate modifications, set up, execute, and
+analyze the new experiments using:
+
+.. code-block:: console
+
+    $ ramble workspace setup
+    $ ramble on
+    $ ramble workspace analyze
+
+**NOTE**: Since you changed the package definition for GROMACS, it will be
+recompiled (unless you compiled it outside of Ramble) during the ``ramble
+workspace setup`` command. This will likely take longer than changing
+experiments and performing setup again.
+
+This creates a ``results`` file in the root of the workspace that contains
+extracted figures of merit. If the experiments were successful, this file will
+show the following results:
+
+* Average Timestep Time: Time (in seconds) on average each timestep takes
+* Cumulative Timestep Time: Time (in seconds) spent executing all timesteps
+* Minimum Timestep Time: Minimum time (in seconds) spent on any one timestep
+* Maximum Timestep Time: Maximum time (in seconds) spent on any one timestep
+* Number of timesteps: Count of total timesteps performed
+* Avg. Max Ratio Time: Ratio of Average Timestep Time and Maximum Timestep Time
```

Comment:
```suggestion
* Avg. Max Ratio Time: Ratio of Average Timestep Time and Maximum Timestep Time

Ramble also supports uploading the analyzed data to online databases, using `ramble workspace analyze --upload`. We will not cover that there, but it is a very useful feature for production experiments
```

Reply:
ie shall we add a plug for zep here? :)

Reply:
Sure, I think we should also add (in the docs) what the supported formats for `--upload` are. I'll take care of adding this into the tutorial, and make a task to add docs around supported upload formats.